### ISSUE TITLE
Test Explorer: Broaden the filters in the tool bar to include Ignored Tests

### ISSUE THREAD

**Issue by Randalliser** [rubberduck-vba/Rubberduck#5466]
**Justification**
As a developer doing unit tests, I would like to only see the unit tests that are subject to my current testing activities. 

When I ignore a test it is because I do not wish to deal with that test right at the moment. Perhaps it is still being coded or perhaps there is a known bug, to which I will return to later when I have a solution. 

**Description**
Add a new filter button next to the other 'outcome' filters called "Ignored", which when toggled, will remove any ignored test from the list.

**Additional context**
![Unit Test Filter Buttons](https://user-images.githubusercontent.com/8215393/79828250-b83c5d00-83e3-11ea-8672-2a8931f21042.png)

**Comment by musharafmaqbool** [comment]
hey i would like to contirbute ....can i take this up?

**Comment by IvenBach** [comment]
Yes. It has both the [good-first-issue] and [up-for-grabs] labels on it.
